kotlin-wrappers-bom/build.gradle.kts

```diff
@@ -1,12 +1,17 @@
 plugins {
     `java-platform`
     `publish-conventions`
+    id("dev.adamko.dokkatoo-html")
 }
 
-dependencies {
-    constraints {
-        for (library in getLibraryProjects(rootProject)) {
-            api(project(library.path))
+configurations.api.configure {
+    // lazily add enabled subprojects to the BOM
+    dependencyConstraints.addAllLater(
+        kotlinWrapperSubprojects.bomDependencies.map { subproject ->
+            logger.info("[$path] adding ${subproject.size} subprojects to BOM: $subproject")
+            subproject.sorted().map { coord ->
+                project.dependencies.constraints.create(project(coord))
+            }
         }
-    }
+    )
 }
```

Comment:
TODO remove kotlinWrapperSubprojects service. It will be reviewed in a separate PR https://github.com/JetBrains/kotlin-wrappers/pull/2379/